modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/frontend/data/set/view/table/NotificationTemplatesTableFDSView.java

```diff
@@ -35,6 +35,10 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"name", "name",
 			fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
 				"actionLink")
+		).add(
+			"system", "source",
+			fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
+				"notificationTemplateSourceDataRenderer")
```

Comment:
Do we need to create a new one? 
Please, see `ListTypeDefinitionsTableFDSView.java`. This class reuses the class implemented in object-web.

Reply:
I talked to @alinedoleron and we conclude that to reuse the [SourceDataRenderer.tsx](https://github.com/liferay/liferay-portal/blob/2cb48648e00740e688570a8aec0738c8f5e502d5/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/SourceDataRenderer.tsx) is a little bit complicated. 

It's necessary to create a reusable component in `object-js-components-web` and use it in the [SourceDataRenderer.tsx](https://github.com/liferay/liferay-portal/blob/2cb48648e00740e688570a8aec0738c8f5e502d5/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/SourceDataRenderer.tsx) and [NotificationTemplateFDSPropsTransformer.tsx](https://github.com/liferay-objects/liferay-portal/blob/ca71006ed6537c03f8cc1deeeaf71cec28edc550/modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/NotificationTemplateFDSPropsTransformer.tsx).

The path for this files is defined [here](https://github.com/liferay-objects/liferay-portal/blob/ca71006ed6537c03f8cc1deeeaf71cec28edc550/modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/NotificationTemplateFDSPropsTransformer.tsx). I believe this path is pointing directly to the module's resources directory.

It seems too work for a small result.

Given this, I'm inclined to go with the approach of create this a new notificationTemplateSourceDataRenderer.

Reply:
Since this component is very simple, let's go with the approach of adding a new component.
However, the guidance is to reuse frontend components and stop creating new ones.